### ISSUE TITLE
TypeError: Cannot read property 'showTimestamps' of undefined

### ISSUE THREAD

**Issue by peZhmanParsaee** [lensapp/lens#2699]
App crash at http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/pods?lens-namespaces=default

**Error stack:**
 TypeError: Cannot read property 'showTimestamps' of undefined
    at KD.get logs (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:8298282)
    at ce (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:12079)
    at e.computeValue (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:18380)
    at e.trackAndCompute (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:18206)
    at e.get (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:17257)
    at e.read (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:51873)
    at KD.get (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:53326)
    at KD.render (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:8298389)
    at Ce (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:13973)
    at http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:7557924

**Component stack:**
    at KD (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:8296029)
    at div
    at tF (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:8302064)
    at div
    at div
    at cF (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:8308047)
    at footer
    at div
    at pU (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:8513336)
    at cc (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:7682615)
    at t (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:223994)
    at BU (http://06a7f306-97a0-4d5f-9191-37e3bb5230e3.localhost:51151/build/Lens.js:2:8514840)

**Comment by jim-docker** [comment]
What version of Lens does this happen on? Does it happen on startup? Could be duplicate of #2652

**Comment by peZhmanParsaee** [comment]
@jim-docker  Lens 4.2.3

**Comment by nevalla** [comment]
My gut feeling is that this error is coming from this line:
https://github.com/lensapp/lens/blob/master/src/renderer/components/dock/log-list.tsx#L104

@aleksfront you know this the best, but would it be safe if we just do 

```typescript
const showTimestamps = logTabStore.getData(this.props.id)?.showTimestamps;
```

Or what could be the case when `logTabStore` is missing data?

**Comment by Nokel81** [comment]
Duplicate of https://github.com/lensapp/lens/issues/2652

Hopefully we can get this fixed for 5.0.0 or 5.0.1
